### ISSUE TITLE
No module named 'tensorflow_text.core'

### ISSUE THREAD

**Issue by xwklwwlf** [tensorflow/text#771]
mac os 12.0.1 python3.9 tensorflow-text 2.7.0
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
/var/folders/4n/spysphms5ljbd93j65p1zk1h0000gn/T/ipykernel_67952/2835017744.py in <module>
----> 1 import tensorflow_text as text

/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/tensorflow_text/__init__.py in <module>
     18 
     19 # pylint: disable=wildcard-import
---> 20 from tensorflow_text.core.pybinds import tflite_registrar
     21 from tensorflow_text.python import keras
     22 from tensorflow_text.python import metrics

ModuleNotFoundError: No module named 'tensorflow_text.core'

![image](https://user-images.githubusercontent.com/4093637/141663891-105fe956-2fbb-4781-957a-de18fcd75e1d.png)

**Comment by Razvanip13** [comment]
I have the same issue

**Comment by leordsong** [comment]
I have the same issue

**Comment by codertimo** [comment]
I have the same issue

**Comment by aidoskanapyanov** [comment]
Me too

**Comment by broken** [comment]
Yes; it looks like the files were not included in the build. We are investigating now.

**Comment by Razvanip13** [comment]
> 
> 
> Yes; it looks like the files were not included in the build. We are investigating now.

Please, keep us tuned cause I'm tired of importing hubs in google colab ;-;. I want my GPU back

**Comment by oxform** [comment]
Same issue aswell

**Comment by broken** [comment]
tl;dr  We released v2.7.2 which you can now use.

This is actually more complicated than leaving out a few files, and we are still working on a solution. In the meantime, we temporarily released v2.7.2 to give users a non-problematic package. This is essentially the 2.6 branch built for TF 2.7. As such, it doesn't contain the new features of 2.7, but it should resolve current headaches of using TF Text with TF 2.7.

**Comment by laszlo-luminous** [comment]
I'm running into the same issue after installing `tensorflow-text-nightly` (only on mac). Would it be possible to apply the same temporary fix to that?

**Comment by broken** [comment]
We finally got this fixed, so tonight's nightly should be okay. Also, the release eng should be pushing out a new v2.7.3 tomorrow.

**Comment by broken** [comment]
v2.7.3 was released this morning. Thank you everybody for your patience. We're discussing how we could implement "package" tests instead of relying on the unit tests that didn't catch this originally to prevent something like this from happening again.

Also, I will remove 2.7.2 next week since it is not a true 2.7 release to prevent future issues. Have a great weekend!
